### DISCUSSION TITLE
Remove usergroup border for Alumni, BN, LVD, BSC in chat

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26045] [Ideas]
They automatically get into a moderation position, which they don't fulfill, but seem as "important" as one. I hope you get me.

IMO they should have normal user colors.

**Comment by bdach** [comment]
> I hope you get me

I do not actually. What does

> They automatically get into a moderation position

mean?

Should these user groups not get coloured usernames on web, too? I'm really confused why this is perceived as an issue.

**Reply by venix12** [reply]
i believe they just mean historically coloured nickname in the chat meant someone with chatmod permissions (not saying it should stay that way, but something to consider i guess)

**Reply by tatitatai** [reply]
yup, which causes anything non-moderators say seem as important as moderator messages (thinking about it doesn't seem that bad but in moderation it kinda is)
